src/components/Link/README.md

```diff
@@ -0,0 +1,192 @@
+<!--GITHUB_BLOCK-->
+
+# Link
+
+<!--/GITHUB_BLOCK-->
+
+```tsx
+import {Link} from '@gravity-ui/uikit';
+```
+
+## Description
+
+Link - is a part of the text, that, when clicked, takes the user to another part of the page, another page inside the service, or to a page of another external service.
+
+The main difference from the [Buttons](/components/uikit/button) is the navigation function. Links more often lead to other pages or open new browser tabs.
```

Comment:
If we reference another component, a good approach is to separate sections of landing and github docs (check the Label docs). So links would work both on GitHub and site. It's a bit duplication, but I'm working on automatic links conversion from `../Button/README.md` to `/components/uikit/button`.

Reply:
@LakeVostok I've recently merged this feature to the landing repo, link can be safely changed to `../Button`

Reply:
Done